optimum/exporters/openvino/model_patcher.py

```diff
@@ -83,10 +82,8 @@ def patch_model_with_bettertransformer(model):
     if hasattr(model, "use_bettertransformer") and model.use_bettertransformer is True:
         return model
 
-    if (
-        getattr(model.config, "model_type") in {"gpt_bigcode", "llama", "gemma"}
-        and is_transformers_version(">=", "4.38")
-        and is_openvino_version("<", "2024.1.0-14612")
+    if getattr(model.config, "model_type") in {"gpt_bigcode", "llama", "gemma"} and is_openvino_version(
+        "<", "2024.1.0-14612"
```

Comment:
todo : do the same for old openvino versions (for < v2025.1.0)

https://github.com/huggingface/optimum-intel/blob/v1.25.2/setup.py#L69